src/webui/api/appcontroller.h

```diff
@@ -40,6 +43,9 @@ class AppController : public APIController
 public:
     using APIController::APIController;
 
+private:
+    QDir::Filters parseDirectoryContentMode(const QString &visibility);
```

Comment:
You don't really need it to be a class member. Just put it as "free" function into unnamed namespace in `appcontroller.cpp`.

Reply:
Done